Packs/Core/ReleaseNotes/1_3_26.md

```diff
@@ -0,0 +1,11 @@
+#### Playbooks
+##### Ransomware Response
+- Added the `get-prevalence` to verdict check conditional task.
```

Comment:
did we add the `get-prevalence` to verdict check conditional task? in the ransom playbook?